one_collect/src/helpers/exporting/os/linux.rs

```diff
@@ -1236,7 +1237,8 @@ impl OSExportMachine {
                     }
 
                     if let Ok(filename) = machine.strings.from_id(map.filename_id()) {
-                        if let Ok(file) = proc.open_file(Path::new(filename)) {
+                        let filename_owned = filename.to_owned();
```

Comment:
Can you move this call into the tracing macro to avoid taking ownership of the filename when logging is disabled?